cypress/integration/userCanCreateTimeRecording.feature.js

```diff
@@ -4,43 +4,50 @@ describe("User can create time recording", () => {
 	it("successfully", () => {
 		cy.server();
 		cy.route({
-			method: 'POST',
-			url: 'https://demo-stable.kimai.org/api/timesheets',
-			response: 'fixtures:save_data.json',
+			method: "POST",
+			url: "https://demo-stable.kimai.org/api/timesheets",
+			response: "fixtures:save_data.json",
 			headers: {
 				"X-AUTH-USER": "susan_super",
 				"X-AUTH-TOKEN": "api_kitten"
 			}
-		})
-		cy.visit("http://localhost:3000")
-		cy.get('input[type=name]').type('anna_admin')
-		cy.get('input[type=password]').type('api_kitten')
-		cy.get('button[type=submit]').click()
-		cy.get('input[name="begin"]').type("07:00")
-		cy.get('input[name="end"]').type("08:00")
-		cy.get('button[name="create"]').click()
-		cy.contains('Your time was saved')
-	})
+		});
+		cy.visit("http://localhost:3000");
+		cy.get("input[type=name]").type("anna_admin");
+		cy.get("input[type=password]").type("api_kitten");
+		cy.get("button[type=submit]").click();
+		cy.get("#customer > .dropdown").click();
+		cy.contains("Hammes-Kilback").click();
+		cy.get("#projects > .dropdown").click();
+		cy.contains("Advanced content-based functionalities").click();
+		cy.get('input[name="begin"]').type("07:00");
+		cy.get('input[name="end"]').type("08:00");
+		cy.get(':nth-child(9) > input').type("25");
+		cy.get('button[name="create"]').click();
+		cy.contains("Your time was saved");
+	});
 
-	it("non successfully", () => {
+	it("unsuccessfully", () => {
 		cy.server();
 		cy.route({
-			method: 'POST',
-			url: 'https://demo-stable.kimai.org/api/timesheets',
-			response: 'fixtures:failed_saving_data.json',
+			method: "POST",
+			url: "https://demo-stable.kimai.org/api/timesheets",
+			response: "fixtures:failed_saving_data.json",
 			headers: {
 				"X-AUTH-USER": "susan_super",
 				"X-AUTH-TOKEN": "api_kitten"
 			},
 			status: 400
-		})
+		});
 		cy.visit("http://localhost:3000");
-		cy.get('input[type=name]').type('anna_admin')
-		cy.get('input[type=password]').type('api_kitten')
-		cy.get('button[type=submit]').click()
+		cy.get("input[type=name]").type("anna_admin");
+		cy.get("input[type=password]").type("api_kitten");
+		cy.get("button[type=submit]").click();
 		cy.get('input[name="begin"]').type("abcd");
 		cy.get('input[name="end"]').type("abcd");
 		cy.get('button[name="create"]').click();
-		cy.contains('Your time was not saved, make sure that you use the correct format')
+		cy.contains(
+			"Your time was not saved, make sure that you use the correct format"
+		);
```

Comment:
I don't mind the cryptic message, but please take a look at my proposed assertion format above.

Reply:
Fixed